assets/prototype/domain/eventEditor/containers/mutations/useCreateDateMutation.js

```diff
@@ -0,0 +1,38 @@
+import gql from "graphql-tag";
+import { useMutation } from "@apollo/react-hooks";
+
+const CREATE_DATE = gql`
+	mutation createDatetime($input: CreateDatetimeInput!) {
+		createDatetime(input: $input) {
+			datetime {
+				id
+				name
+				description
+				sold
+				reserved
+				order
+				start
+				end
+				length
+				startDate
+				endDate
+				startTime
+				endTime
+				capacity
+				isPrimary
+				isSoldOut
+				isUpcoming
+				isActive
+				isExpired
+			}
+		}
+	}
+`;
+
+const useCreateDateMutation = () => {
+	const [createDate, { data }] = useMutation(CREATE_DATE);
+
+	return [createDate, { data }];
```

Comment:
Why not `return useMutation(CREATE_DATE);`?

Reply:
That is a good question.
I was thinking that if we would like or not to add some checks here before it goes into the components.
But, I think we can get back on this later after the proto is finished.